src/main/java/frc/robot/RobotContainer.kt

```diff
@@ -17,6 +20,9 @@ class RobotContainer {
     // The robot's subsystems and commands are defined here...
     private val m_exampleSubsystem = ExampleSubsystem()
     private val m_autoCommand = ExampleCommand(m_exampleSubsystem)
+    val elbow_motor = CANSparkMax(Constants.elbowmotorID, CANSparkMaxLowLevel.MotorType.kBrushless)
+    val elevator_motor = CANSparkMax(Constants.elevatormotorID, CANSparkMaxLowLevel.MotorType.kBrushless)
```

Comment:
See comments below. Remove the cansparkmax initializations, pass in numbers corresponding to CAN ID for the arm subsystem.